lib/ohai/plugins/packages.rb

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
shouldn't this be encoding?

Reply:
unnecessary now (this is pre-2.0 i think)

Reply:
Don't you still want it if you're trying to force the encoding.

Reply:
only if you want to set it to something other than utf-8

Reply:
Did a `(setq ruby-insert-encoding-magic-comment nil)` and cleaned this up. Sorry.